inference-engine/tests/unit/gna/ngraph/transformations/handle_transposes_around_matmul.cpp

```diff
@@ -235,19 +259,21 @@ TEST(TransformationTests, RemoveTransposeBeforeMatmulTestReshapeInOutEq) {
 }
 
 TEST(TransformationTests, InsertTransposeAfterMatmulTest) {
-    for (auto enable_add : { true, false}) {
-        for (auto matmul_on_left_side : { true, false}) {
-            for (auto enable_fq : { true, false}) {
-                RunTest(
-                    handle_transpose_after_matmul::CreateMatmulFunction(
-                        {4, 1}, {1, 8}, {2, 16}, false, true, enable_add, matmul_on_left_side, enable_fq),
-                    handle_transpose_after_matmul::CreateMatmulTransposeFunction(
-                        {4, 1}, {1, 8}, {2, 16}, true, true, enable_add, matmul_on_left_side, enable_fq));
-                RunTest(
-                    handle_transpose_after_matmul::CreateMatmulFunction(
-                        {1, 256}, {256, 256}, {8, 32}, false, true, enable_add, matmul_on_left_side, enable_fq),
-                    handle_transpose_after_matmul::CreateMatmulFunction(
-                        {1, 256}, {256, 256}, {8, 32}, false, true, enable_add, matmul_on_left_side, enable_fq));
+    for (auto enable_add : { true, false }) {
+        for (auto matmul_on_left_side : { true, false }) {
+            for (auto enable_fq1 : { true, false }) {
+                for (auto enable_fq2 : { true, false }) {
```

Comment:
Why not creating separate tests with params

Reply:
I don't see a point... This just covers all cases.